README.md

```diff
@@ -23,80 +46,61 @@ Primeiramente, clone o repositório para sua máquina local usando o comando:
 git clone <Link para o repositório>
 ```
 
-Para ter seu chatbot Rasa funcionando, certifique-se de estar dentro da pasta do projeto e então execute no terminal o seguinte comando:
+Para ter seu chatbot funcionando, certifique-se de estar dentro da pasta do projeto e então execute no terminal o seguinte comando:
 
 ```sh
-make first-run
+sudo make first-run
 ```
 
-⚠️ **Atenção**: Caso ocorra algum erro de permissão, executar o comando `sudo make first-run`.
+⚠️ **Dica**: Se você já tiver permissão de sudo por padrão, pode executar os comandos diretamente, como `make first-run`.
 
 
-Esse comando irá construir a infraestrutura necessária (subir containers com as dependências, treinar o chatbot, etc) para possibilitar a interação com o chatbot.
+Esse comando irá construir a infraestrutura necessária (subir contêineres com as dependências, treinar o chatbot, etc) para possibilitar a interação com o chatbot.
 
 Tudo está dockerizado então você não deve ter problemas de instalação do ambiente.
 
-Depois que tudo for instalado, você verá a seguinte mensagem e pode começar a interagir com o bot
+Depois que tudo for instalado, você verá a seguinte mensagem:
 
 ```sh
 Bot loaded. Type a message and press enter (use '/stop' to exit):
 Your input ->
 ```
 
-Para fechar a interação com o bot é só dar `ctrl+c`.
+Agora você pode começar a interagir com o bot!
 
+Para fechar a interação com o bot é só dar `ctrl+c`.
```

Comment:
o `ctrl+c` é um comando de forçar a parada. O ideal é colocar o `/stop` que é o comando do próprio bot para encerrar de uma forma menos forçada a execução do bot.

```suggestion
Para fechar a interação com o bot, você pode mandar a mensagem `/stop` ou caso ele tenha travado, pode forçar a parada com `ctrl+c`.
```